.gitignore

```diff
@@ -17,6 +17,7 @@ logs
 out
 target
 tmp
+tools
```

Comment:
When we run the build step in TC, I created the tools directory via `build-tc` script. That directory contains the yarn tool in TC. If a developer runs that script locally, we don't want to commit that folder to the repo.